### PR TITLE
dbus: add mask/unmask for unit files

### PR DESCRIPTION
**Pull request by jkahrs**
add methods to trigger the masking and unmasking of unit files

### CODE REVIEW

**Comment by lucab** [comment]
ok to test

File: dbus/methods_test.go [hunk 1]
```diff
@@ -507,3 +507,56 @@ func TestConnJobListener(t *testing.T) {
...
+	runPath := filepath.Join("/run/systemd/system/", target)
+
+	// 1. Enable the unit
+	install, changes, err := conn.EnableUnitFiles([]string{abs}, true, true)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if install != false {
+		t.Log("Install was true")
```
Comment (lucab) [inline dbus/methods_test.go]: For reference, this is due to a semantic break after https://github.com/systemd/systemd/pull/3016

File: dbus/methods.go [hunk 2]
```diff
@@ -473,6 +473,84 @@ type DisableUnitFileChange struct {
...
 }
 
+// MaskUnitFiles() may be used to mask one or more units in the system
+//
+// It takes a list of unit files to mask (either just file names or full
+// absolute paths if the unit files are residing outside the usual unit
+// search paths), and two booleans:
+// runtime: to specify whether the unit was enabled for runtime only
+// (true, /run), or persistently (false, /etc).
+// and force
```
Comment (lucab) [inline dbus/methods.go]: The list of parameters is a bit awkward to read (it took me a couple of seconds to understand the dangling "and force"). Can you reword the last paragraph as a bullet list like:
```
It takes three arguments:
 * list of unit files to mask, either...
 * runtime flag to specify...
 * force flag
```

File: dbus/methods.go [hunk 3]
```diff
@@ -473,6 +473,84 @@ type DisableUnitFileChange struct {
 	Destination string // Destination of the symlink
 }
 
+// MaskUnitFiles() may be used to mask one or more units in the system
```
Comment (lucab) [inline dbus/methods.go]: You can remove the parenthesis and the modal form:
```
MaskUnitFiles masks one or...
```

File: dbus/methods.go [hunk 4]
```diff
@@ -473,6 +473,84 @@ type DisableUnitFileChange struct {
...
+	Filename    string // File name of the symlink
+	Destination string // Destination of the symlink
+}
+
+// UnmaskUnitFiles() may be used to unmask one or more units in the system
+//
+// It takes a list of unit files to mask (either just file names or full
+// absolute paths if the unit files are residing outside the usual unit
+// search paths), and one boolean: whether the unit was enabled for runtime
+// only (true, /run), or persistently (false, /etc).
```
Comment (lucab) [inline dbus/methods.go]: Same remarks as above (just for uniformity).

**Comment by lucab** [comment]
Thanks @jkahrs, super nice contribution! Code and test are ok, just a minor remark on the doc.

**Comment by jkahrs** [comment]
Thank you @lucab I'll update the PR shortly

**Comment by lucab** [comment]
@jkahrs I see you update the PR, seems fine now :+1:. Can you please just squash this into a single commit?

**Comment by jkahrs** [comment]
@lucab  no problem. Not sure why debian broke though.

**Comment by lucab** [comment]
@jkahrs It looks like a temporary glitch in travis builder. I've re-triggered it and everything is back to normality. LGTM.

**Pull request merged**